### DISCUSSION TITLE
Mount ext4-image from sd-card

### DISCUSSION THREAD

**Discussion by RoseDeSable** [termux/termux-app#3800] [Q&A] [answered]
Hi,
 the filesystem on the extended storage is from type fat, but I need there an ext4. So I want to create a writable ext4-image there. I mean, that I want to create a file, that simulate inside an ext4-filesystem. Can termux mount it as ext4 ?

**Comment by twaik** [comment, marked as answer]
You can use busybox's `mount` applet or `mount` tool from `mount-utils` package. Both require root in order to work.

**Reply by twaik** [reply]
Both busybox and mount (from mount-utils) are available for installation.

**Reply by RoseDeSable** [reply]
Ok,
 termux run as no root under android. Need I root rights under android or only under termux ?

**Reply by twaik** [reply]
termux _inherits_ root (su) of your android system. And it is **required** for mounting operations.

**Reply by RoseDeSable** [reply]
Ok,
 I want to try it. I use a Samsung tablet Galaxy A8 with the android version 13 (One UI Core 5.1.1). I will show, if I can commands with root-rights. My experience in root-things isn't good under this device type.

**Reply by twaik** [reply]
It does not matter what device or Android version you have. Both busybox and mount-utils have the same behaviour on all linux distros.

**Reply by RoseDeSable** [reply]
Hi,
 my android has no root rights. Therefore termux cannot inherit them. I installed the tool fakeroot in termux. But mounting the image isn't possible, because no loop device is active (/dev/loopx). What can I do now ?

**Reply by agnostic-apollo** [reply]
There is no other way than rooting the device for real. fakeroot will not work.

**Reply by RoseDeSable** [reply]
Ohh,
 thank you for the discussion, although the result isn't satisfactorily. It is dangerous to try to root a Samsung device by special root images. The security design of the tablet is built in the manner, that a special electronic circuit of the hardware will be burnt. Only Samung itself can repair it.
 How can I close the discussion ?

Best Regards
Rose

**Comment by agnostic-apollo** [comment]
https://github.com/mpartel/bindfs/issues/124#issuecomment-1368201418
